README.md

```diff
@@ -397,17 +402,18 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Vinyl Music Player**](https://github.com/AdrienPoupa/VinylMusicPlayer) <sup>**[[F-Droid](https://f-droid.org/app/com.poupa.vinylmusicplayer)]**</sup>
 * [**Music Player GO**](https://github.com/enricocid/Music-Player-GO) <sup>**[[F-Droid](https://f-droid.org/app/com.iven.musicplayergo)]**</sup>
 * [**Odeon**](https://github.com/thibseisel/android-odeon) <sup>**[[F-Droid](https://f-droid.org/app/fr.nihilus.music)]**</sup>
-* [**Metro**](https://github.com/MuntashirAkon/Metro) <sup>**[[F-Droid](https://f-droid.org/app/io.github.muntashirakon.Music)]**</sup>
+* [**Retro Music Player**](https://github.com/RetroMusicPlayer/RetroMusicPlayer)
```

Comment:
This was unlisted since it contains proprietary components unfortunately (https://github.com/RetroMusicPlayer/RetroMusicPlayer/issues/59, https://github.com/RetroMusicPlayer/RetroMusicPlayer/issues/1093)